cdc/processor/processor.go

```diff
@@ -144,14 +145,7 @@ func (p *processor) AddTableSpan(
 			// be stopped on original capture already, it's safe to start replicating data now.
 			if !isPrepare {
 				if p.redo.r.Enabled() {
-					var redoResolvedTs model.Ts
-					if barrier != nil {
-						redoResolvedTs = barrier.GlobalBarrierTs
-					} else {
-						stats := p.sinkManager.r.GetTableStats(span)
-						redoResolvedTs = stats.BarrierTs
-					}
-					p.redo.r.StartTable(span, redoResolvedTs)
+					p.redo.r.StartTable(span, checkpoint.ResolvedTs)
```

Comment:
please add a comment to tell why we should dou this

Reply:
Done, PTAL